proto/osmosis/twap/v1beta1/twap_record.proto

```diff
@@ -37,20 +37,20 @@ message TwapRecord {
 
   // We store the last spot prices in the struct, so that we can interpolate
   // accumulator values for times between when accumulator records are stored.
-  string p0_last_spot_price = 6 [
```

Comment:
Change of state entry names to avoid secondary migrations in upgrade

Reply:
Niec, this LGTM